bin/virtualenv_test.sh

```diff
@@ -15,8 +15,19 @@ set -euo pipefail
         echo "black --target-version py37 --check TWLight"
         if black --target-version py37 --check TWLight
         then
+            echo "${TWLIGHT_HOME}/tests/shunit/twlight_i18n_lint_test.sh"
+            ${TWLIGHT_HOME}/tests/shunit/twlight_i18n_lint_test.sh
+
+            # https://phabricator.wikimedia.org/T255167
```

Comment:
We might want to link to https://github.com/WikipediaLibrary/TWLight/wiki/Translation instead of this Phab task?

Reply:
agreed